code/code/disc/disc_shaman_armadillo.cc

```diff
@@ -421,8 +429,7 @@ int celerite(TBeing *caster, TBeing *victim, int level, short bKnown)
     aff.type = SPELL_CELERITE;
     aff.level = level;
     aff.duration = caster->durationModify(SPELL_CELERITE, (aff.level / 3) * Pulse::UPDATES_PER_MUDHOUR);
-    aff.modifier = -100;
-    aff.location = APPLY_ARMOR;
+    aff.location = APPLY_NONE;
```

Comment:
so what'll it do now?

Reply:
Reduces command lockout and does haste 
https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/utility.cc#L1569
https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/offense.cc#L2260

Reply:
Since Shaman don't wear armor I guess the coder added some extra AC onto their spells. But now we have shiny silver potions and everyone has these spells on at all times. This just brings them in line with mage accelerate and haste.